components/PostReadingTime.js

```diff
@@ -2,7 +2,7 @@ import { calculateReadingTime } from 'utils/helpers'
 
 export default function PostReadingTime({ blocks }) {
   const words = blocks.reduce((words, block, i) => {
-    block.html.trim().split(' ').forEach(word => word && words.push(word))
+    if(block.tag!=="code") block.html.trim().split(' ').forEach(word => word && words.push(word))
```

Comment:
Please keep the style guide.
Space between the if and "(", and also between expressions and operators